merlin/dag/ops/add_metadata.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from merlin.core.protocols import Transformable
+from merlin.dag.base_operator import BaseOperator
+from merlin.dag.selector import ColumnSelector
+from merlin.schema.tags import Tags
+
+
+class AddMetadata(BaseOperator):
+    """
+    This operator will add user defined tags and properties
+    to a Schema.
+    """
+
+    def __init__(self, tags=None, properties=None):
+        super().__init__()
+        self.tags = tags or []
+        self.properties = properties or {}
+
+    def transform(
```

Comment:
This appears to be the default implementation of `transform` in the `BaseOperator`. Could this be omitted or, are we adding this here as part of a convention of adding a transform for every operator?